pkg/resource/feature_group/testdata/feature_group/cr/fg_cmd_invalid_create_attempted.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
```

Comment:
rename to *_create_failed

why do these files have fg_cmd prefix?

Reply:
I was following the naming conventions set by the elasticache controller [here](https://github.com/aws-controllers-k8s/elasticache-controller/tree/main/pkg/resource/replication_group/testdata/replication_group/cr). Where they list the replication group testing files as always prefixed by rg_cmd (where rg stands for replication group).

The equivalent of this invalid input test in the elasticache-controller is names as [rg_cmd_invalid_create_attempted.yaml](https://github.com/aws-controllers-k8s/elasticache-controller/blob/main/pkg/resource/replication_group/testdata/replication_group/cr/rg_cmd_invalid_create_attempted.yaml). Therefore I was using that naming convention here. But if you feel the name is not clear enough, we can change this naming convention in our resource.

If you would like, I can go ahead and remove the prefix from this resource and the other resources in PRs #68 and #76

Reply:
@BriannaRoskind "cmd_" stands for "Cluster-mode Disabled" in Elasticache's Replication Groups :) You can safely remove that part of the string here.

Reply:
Thanks for clarifying :) I will apply this change to this PR as well as PR's #68 and #76.